common/src/main/scala/rpgboss/player/InputMultiplexer.scala

```diff
@@ -131,6 +139,10 @@ class InputMultiplexer extends InputAdapter {
     case Keys.M => Some(MyKeys.Cancel)
     case Keys.X => Some(MyKeys.Cancel)
     case Keys.ESCAPE => Some(MyKeys.Cancel)
+    case Keys.A => Some(MyKeys.A)
```

Comment:
Change the default mappings to something more accessible. Maybe Ctrl, Shift, Alt, Enter?